Source/WebCore/Modules/geolocation/Geolocation.cpp

```diff
@@ -441,7 +448,7 @@ void Geolocation::makeCachedPositionCallbacks()
     for (auto& notifier : m_requestsAwaitingCachedPosition) {
         // FIXME: This seems wrong, since makeCachedPositionCallbacks() is called in a branch where
         // lastPosition() is known to be null in Geolocation::setIsAllowed().
-        notifier->runSuccessCallback(lastPosition());
+        notifier->runSuccessCallback(RefPtr { lastPosition() }.get());
```

Comment:
Maybe we should cache `lastPosition()` before the loop to avoid ref-counting churn at each loop iteration.